### ISSUE TITLE
bigip_ssl_(key|key_cert|certificate): Modules enforces '.key' and '.crt' extension while F5 BIG-IP TMOS doesn't

### ISSUE THREAD

**Issue by simonkowallik** [F5Networks/f5-ansible#2306]
##### COMPONENT NAME

```
bigip_ssl_certificate
bigip_ssl_key_cert
bigip_ssl_key
```

version: 1.22.1

##### ANSIBLE VERSION

```
currently latest
```
##### BIGIP VERSION

```
16.1.3.3
```
##### SUMMARY

The ansible module enforces '.key' and '.crt' extension for keys and certificates.

This is related to: https://github.com/F5Networks/f5-ansible-bigip/issues/58

While adding extensions could be considered a good practice, F5 BIG-IP TMOS (WebUI, tmsh, iControl REST API) does not enforce adding extensions to certificate or key files. Therefore the ansible module is not consistent with the features and behaviour of TMOS.

This is a problem because it prevents using ansible to manage existing sets of certificates and keys as well as reference issuer certificates that do not end with '.crt'.

##### STEPS TO REPRODUCE

```
- name: Import both key and cert
  bigip_ssl_key_cert:
    key_content: "{{ lookup('file', 'key.pem') }}"
    key_name: cert1
    cert_content: "{{ lookup('file', 'cert.pem') }}"
    cert_name: cert1
    state: present
    provider:
      password: secret
      server: lb.mydomain.com
      user: admin
  delegate_to: localhost
```

This will produce `sys file ssl-cert cert1.crt` and `sys file ssl-key cert1.key` on the BIG-IP. `.crt` and `.key` are added even though not specified in the playbook (`cert_name` and `key_name`).

##### EXPECTED RESULTS

The expectation is to create the objects with the specified name without modification.

##### ACTUAL RESULTS

The produced objects are always named with `.crt` and `.key` file extensions.

**Comment by KrithikaChidambaram** [comment]
Hi, tracking this internally INFRAANO-1118, thanks!

**Comment by urohit011** [comment]
This one was closed  in #2318
